jetty-servlet-api/pom.xml

```diff
@@ -131,38 +142,44 @@
                     <exclude>META-INF/MANIFEST.MF</exclude>
                   </excludes>
                 </filter>
-               </filters>
-               <transformers>
-                <transformer implementation="org.apache.maven.plugins.shade.resource.IncludeResourceTransformer">
+              </filters>
+              <transformers>
+                <transformer
+                  implementation="org.apache.maven.plugins.shade.resource.IncludeResourceTransformer">
                   <resource>META-INF/MANIFEST.MF</resource>
                   <file>${project.build.outputDirectory}/META-INF/MANIFEST.MF</file>
                 </transformer>
-                 <transformer implementation="org.eclipse.jetty.toolchain.shade.MergeFilesTransformer">
-                   <resource>META-INF/LICENSE</resource>
-                   <files>
-                     <file>${project.build.directory}/sources/META-INF/LICENSE</file>
-                     <file>${project.build.directory}/sources/META-INF/LICENSE.txt</file>
-                   </files>
-                 </transformer>
+                <!-- NOT SURE WE ARE ALLOWED TO ALTER THE LICENSE OF THESE FILES
```

Comment:
we do not alter we only merge to a single license file.

Reply:
"alter" in this sense is us taking the "Eclipse Jetty" licenses and using those for the new artifact, NOT using the ones from `javax-servlet-api`

Reply:
@olamy @joakime the problem here is that the LICENSE for javax.servlet-api is CDDL and the LICENSE for the jetty-schemas is Eclipse. So the final jar containing both artifacts needs to have both licenses in it.